vendor/modules.txt

```diff
@@ -114,6 +114,7 @@ github.com/aws/aws-sdk-go/service/elbv2
 github.com/aws/aws-sdk-go/service/elbv2/elbv2iface
 github.com/aws/aws-sdk-go/service/iam
 github.com/aws/aws-sdk-go/service/iam/iamiface
+github.com/aws/aws-sdk-go/service/kinesis
```

Comment:
Expected to see a go.mod/go.sum update with this.

Reply:
only another module of `github.com/aws/aws-sdk-go/service/` was included - others were already present, so no go.mod or go.sum changes needed (or were caught as a part of `make vendor`)